spectre-meltdown-checker.sh

```diff
@@ -3916,7 +4060,6 @@ check_CVE_2017_5715_linux()
 
 		# if we arrive here and didn't already call pvulnstatus, then it's VULN, let's explain why
 		if [ "$pvulnstatus_last_cve" != "$cve" ]; then
-			# explain what's needed for this CPU
```

Comment:
?

Reply:
Probably also this should go in a different commit

Reply:
That was an ooops. Great catch.It shouldn't have been part of this (was an original prototype).